travel_platform/izitravel_tests.py

```diff
@@ -15,8 +15,7 @@ def setUp(self):
     def test_get_city_by_name_returns_city(self):
         result = self.client.get_city_by_name(self.city)
         keys = ["title", "location", "map", "children_count", "city_uuid"]
-        for key in keys:
-            self.assertIn(key, result)
+        self.assertTrue(all([key in result for key in keys]))
```

Comment:
Well, it may be replaced as it was, if it gives better result.
I proposed this way, maybe it will be useful for you in other cases.